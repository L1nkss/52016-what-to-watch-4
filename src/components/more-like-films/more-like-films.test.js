import MoreLikeFilms from "./more-like-films";

describe(`Testing More like films component`, () => {
  it(`Component should successfully rendered`, () => {
    const tree = renderer.create(<MoreLikeFilms />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
