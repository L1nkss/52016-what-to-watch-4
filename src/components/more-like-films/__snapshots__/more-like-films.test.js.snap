// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing More like films component Component should successfully rendered 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        className="player__video"
        muted={true}
        poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/film-detail/1"
        onClick={[Function]}
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        className="player__video"
        muted={true}
        poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/film-detail/2"
        onClick={[Function]}
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>
</div>
`;
